assesspy/ci.py

```diff
@@ -60,13 +61,13 @@ def boot_ci(
         raise ValueError("'nboot' must be a positive integer greater than 0.")
     check_inputs(estimate, sale_price)
     df = pd.DataFrame({"estimate": estimate, "sale_price": sale_price})
-    n: int = df.size
+    n: int = len(df)
```

Comment:
`df.size` is the length of _all_ columns in the DataFrame, not the number of rows 🤦

Reply:
My bad for missing this as well!